packages/generator/test/gulpfile-helpers.test.ts

```diff
@@ -0,0 +1,201 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import * as diffProcessor from "@commerce-apps/raml-toolkit";
+import { NodeDiff } from "@commerce-apps/raml-toolkit";
+import { diffNewAndArchivedRamlFiles, RamlDiff } from "../src/gulpfileHelpers";
+import { generatorLogger } from "../src/logger";
+
+import _ from "lodash";
+import chai from "chai";
+import chaiAsPromised from "chai-as-promised";
```

Comment:
```suggestion
import { expect } from "chai";
```
`chai-as-promised` isn't used in this file. As a result, we don't need to call `chai.use`, so we don't need to import all of `chai`, just `expect`.